### ISSUE TITLE
CLI: `short-url:visits` exports different columns than web client

### ISSUE THREAD

**Issue by UCI-MIND-IT** [shlinkio/shlink#2311]
### Shlink version

4.3.1

### PHP version

8.2.26

### How do you serve Shlink

Self-hosted Apache

### Database engine

MariaDB

### Database version

10.11.6

### Current behavior

The CLI command `short-url:visits` does not return the same columns as the web client's "Export" feature (**Shlink web client version 4.3.0**):

![Image](https://github.com/user-attachments/assets/8ee41729-b9ae-495e-b939-a11a729b10b5)

Columns returned from the CLI: `"Referer","Date","User agent","Country","City"`

Columns returned from the web client's CSV export: `"date","potentialBot","userAgent","os","browser","referer","country","city","latitude","longitude","visitedUrl","type"`.

We checked the [Shlink CLI documentation](https://shlink.io/documentation/command-line-interface/entry-point/) and the command's help text using the `-h` flag, but couldn't find any information on configuring the types (columns) of visit data to export. Both our Shlink database server and self-hosted Shlink web client server are running on the same Linux distro using Apache.

### Expected behavior

It would be ideal if data exports using shell commands (individually or automated via shell scripts) could provide at least the same parity of data as the web client's "Export" feature.

### Minimum steps to reproduce

1. Create a short link via any method
2. Generate visit stats for this short link by visiting it several times
3. On the Shlink database server, use the CLI to produce the stats for this short link: `/path/to/shlink/bin/cli short-url:visits testshlink > out.txt`
4. On the web client, create an export of the same short link's stats (see screenshot in "Current behavior")
5. Observe that the columns from the CLI export (`head out.txt`) are different from the columns in the web client's export

**Comment by UCI-MIND-IT** [comment]
Thank you for adding CSV export! This will save a step in [converting the table output](https://github.com/shlinkio/shlink/discussions/2312).

We were running a campaign using a few hundred shortlinks and we wanted to export a rich dataset so we could analyze how our shortlinks performed. Specifically, the columns "potentialBot", "os", and "browser" (even though "os" and "browser" could potentially be inferred from some User Agent strings) would have been good information to provide to our team as potential variables in their analysis. The study has already ended, however, and they were able to work with the limited column set.

I agree that allowing configurable fields in CLI exports would be a lot of effort for a smaller advantage, but changing the default fields exported from the CLI to match the fields from frontend exports would be sufficient.

**Comment by acelaya** [comment]
So basically, you were not asking for the columns to match because you were trying to cross-reference those two, but asking for the CLI to include more information that is available via the API. Is that correct? 

If that's the case, that totally makes sense. I think the limited amount of columns is just due to historical reasons, not something consciously decided.

Expanding the list of columns unconditionally, should be doable in a maintainable way.

**Comment by UCI-MIND-IT** [comment]
Yes, that is correct! We trust that both the frontend's API call and backend's CLI command query the database in similar ways, so we wouldn't have cross-referenced the two exports.
